tools/schemacode/bidsschematools/render/text.py

```diff
@@ -140,10 +140,14 @@ def make_glossary(schema, src_path=None):
     for obj_key in sorted(all_objects.keys()):
         obj = all_objects[obj_key]
         obj_marker = obj["key"]
-        obj_def = obj["definition"]
+        obj_def = obj.get("definition", None)
+        if obj_def is None:
```

Comment:
I have add to add this in order to get more meaningful error messages to track down what was causing the tests to fail.

This was due to some 'unindented' part of the yml.

I don't think that this belong in this PR though.

@tsalo @effigies what do you think?

Reply:
I would agree though that this fix/change should not be included in this PR.

Reply:
@Remi-Gau feel free to pull this change out of this PR to a separate PR, we can merge it and then merge master into **this** PR.